optimism/src/keccak/witness.rs

```diff
@@ -159,19 +174,19 @@ impl<F: Field> KeccakInterpreter<F> for Env<F> {
     /////////////////////////
 
     fn mode_absorb(&self) -> Self::Variable {
-        self.variable(KeccakColumn::Selector(Absorb))
+        self.variable(KeccakColumn::Selector(Sponge(Absorb(Middle))))
     }
     fn mode_squeeze(&self) -> Self::Variable {
-        self.variable(KeccakColumn::Selector(Squeeze))
+        self.variable(KeccakColumn::Selector(Sponge(Squeeze)))
     }
     fn mode_root(&self) -> Self::Variable {
-        self.variable(KeccakColumn::Selector(Root))
+        self.variable(KeccakColumn::Selector(Sponge(Absorb(First))))
     }
     fn mode_pad(&self) -> Self::Variable {
-        self.variable(KeccakColumn::Selector(Pad))
+        self.variable(KeccakColumn::Selector(Sponge(Absorb(Last))))
     }
     fn mode_rootpad(&self) -> Self::Variable {
```

Comment:
Is this renamed into `mode_only` in some following commits?

Reply:
No, those modes remain as rootpad, pad, root, etc.